### DISCUSSION TITLE
Display latest updated or created pages on homepage?

### DISCUSSION THREAD

**Discussion by dejantep** [squidfunk/mkdocs-material#6301] [General]
Is there a built-in function that can display latest updated or created pages  on homepage? I'm not referring to blog functionality but native theme functionality. It would raise user awareness about content.

**Comment by alexvoss** [comment]
You can use the [git-revision-date-localized-plugin](https://timvink.github.io/mkdocs-git-revision-date-localized-plugin) and build on [this example of producing a list of the last 10 updated pages](https://timvink.github.io/mkdocs-git-revision-date-localized-plugin/howto/override-a-theme/#example-list-last-updated-pages)

**Reply by dejantep** [reply]
thank you. i'll take look into it.
